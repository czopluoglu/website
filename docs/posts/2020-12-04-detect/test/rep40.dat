10111111001001101110
11111011101011011111
01000000010000000011
11000100101100100110
10101011011011100010
00100100001010100111
10001000011110001111
00111011001100111110
00101000100010101010
11111111111000010110
11100001101010100110
01000001111001100111
11111110111100101111
00001000000011100110
11000101011111101111
10110001011110101011
11000101011110111111
10011000110001101111
11001000111011101101
10010100111001111011
10101110111011011110
10111111110010011110
00000101101110110111
00100001100011100110
00100011111011011010
00101110000100110111
10111000111110100110
01100111001110111111
01010001111010110111
11001100001000100011
11100001110000011110
10100101111101101111
00000100101111101010
00110010100010100001
11101011011001111010
00000001111110110001
01001000000000000110
10101000110011010110
10101001001111111110
10111000111111101101
00011001101110101111
10100101001010101111
11111001011011111001
01111111010000000011
00010011001110101011
00011000001010100101
10110111001000100111
01100101001100111111
10000001001001101110
00101000101100000010
00000001101110010010
00011010111010110111
00110101110110001010
01000001001000111100
10010001011000010111
10001001001000001000
11010001010101110111
10011111101001111011
01110101110111111010
01010001011100001111
10000011011000111110
11110011101001111110
11011010010110000101
01100001011111111111
00000100000000100010
10111001101010000011
00101000011010000111
10010001001110111110
00010101110011001011
11000000001010100111
11110111101011011000
11011010111010011111
00001010001101111010
10111000101000100111
10001010010000100111
11110111100101111011
11011001010000000001
11100011111011111111
10010010111100000110
01101101111110111110
10000000001110101000
11011111011100110111
00100100011100111000
10110101101111101011
11111000011100000011
10010001001010100100
01010000001101011110
11010110101001010000
10101010001101010011
11111010111101100001
11111100000000111101
01011101111011001110
11001011110110111111
00100001100110110110
00101001001111011010
10001111001110100111
00000100101010110100
01010001101010100111
11000000011001111110
11011110111111111111
01110001001001111111
10001001010010001100
11111111100010101110
00101000111110010011
10010001011000101111
11001100101110100011
10001001111000000110
01101001101111111111
10001001111110001000
10011111011011101011
10000101001001001111
11100101101010001111
11101101010111111111
11111010111111111111
11101001011110110111
10110101111100110110
11011111110101100101
11110111111101011111
11100010001010010010
01001101101111110110
10000000010001010000
11100100000110111011
10010001010011010110
11110011011100001110
01101100010010111111
00100001001010010001
01000111001111000010
00000000011111000010
10010111011010000000
10100001000111100011
00100101101111110111
10011001001110111110
00001000100111100011
11011111010000000011
11110100111110001111
01101000111110100010
00011001111111101110
00111011111111111110
10000001110100100110
10011101001110110110
00100111110010000110
10001001000111000001
11010011110100000110
11101001010111110010
10111101110010100111
11000011001100100011
10110111111111000010
10100101011110101111
10000001001110001010
10011110111011001111
01000000001011111111
11110111111111111111
01101111111111110111
00100101001101111111
00011001000011101110
11111011011011111101
11111101001110011010
00001100110111111111
00100001000000101110
10010110011001101111
00011001010101111111
10010011111000000010
01100111101101101011
10001110101100000110
11001001011110000010
10010111001011111111
00101000111110010011
10011010111110010111
11001111111011101100
10111001110000100111
01000100110100011111
01001000001000001011
11111101111110111000
00100101000010010111
10100100110101100000
00100111101000101110
10010111111111111111
11111111110111100011
10110010010001000111
01110111110101100001
10110111101101000100
11011111101010011010
11000001111000101010
01101101011010000011
00011000011001101010
11100101010011110101
01111011111001100111
00110101111111000111
00011001010011001011
11011111101111010110
11011001010000101000
00000001011111111111
11100111111010111110
11111001111010110010
10001010100000111100
01001000000000110110
00010001010000101110
11000111011110101101
10010111110110100010
10100101110000100111
11000000001101001110
11100101101011000111
01000000110010101111
11001000010111110001
11000000101110100111
11111101101110011111
00100000111101010110
11101011001010111101
11100001010110100010
11000110101101101100
10101011111110101111
11000001101010111111
00100000001001100101
00000101111011100111
00110101101110111011
11001001111010100010
10010001000010110010
01001001001010000110
00000001011001110100
01011111010010111110
00101000010011000110
00001001100010000110
00101011101010100011
01101101111011111011
00101001011111001011
01001001001110111100
10101100011010100111
01010101101110000110
11100111111111100111
00001001100111011001
10101000001000000111
11110001101000011101
01010001100010010111
01100100001000000100
11000110010100100111
11000000100010010011
11000100001000100101
00101111011011000010
11000000011110111110
11111101001001000111
11010000111011111010
10100010011111100000
00000100100111100001
00010110101111101110
01101001001110011100
01001101001010100010
11001110001010101011
00010010001011101111
11110001111010101010
10110001011101101111
