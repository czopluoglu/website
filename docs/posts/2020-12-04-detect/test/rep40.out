-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep40.dat                                          

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.40

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.29
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7583
                   IDN index value:   0.7526
                           Ratio r:   0.6219

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    3    1    1    3    1    3    3    1
    2    3    3    4    2    2    2    2    3    4

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    4    5    7   10

   -----------CLUSTER  2-------------

    2   11   15   16   17   18

   -----------CLUSTER  3-------------

    3    6    8    9   12   13   19

   -----------CLUSTER  4-------------

   14   20
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d++++++-------------
   +d+++----+---++-----
   ++d++-----------+---
   +++d+-+-----+-+---+-
   ++++d------+--++--+-
   +----d+-++--++-----+
   +--+-+d--++----+++--
   -------d+++----+--++
   -----+-+d+++-+-++-+-
   -+---++++d+--+----+-
   ------++++d------+-+
   ----+---+--d++++-+++
   ---+-+-----+d+++-+++
   -+---+--++-++d--++--
   -+-++------++-d+++--
   ----+-+++--++-+d+-+-
   --+---+-+----+++d+--
   ------+---+++++-+d-+
   ---++--+++-++--+--d+
   -----+-+--+++----++d
 
 No cross validation for this DETECT run
